src/PrestaShopBundle/Resources/views/Admin/Product/CatalogPage/Lists/products_table.html.twig

```diff
@@ -186,7 +186,7 @@
           {% if has_category_filter == true %}
             <td>
               {% if not(activate_drag_and_drop) %}
-                <input type="button" class="btn btn-outline-secondary" name="products_filter_position_asc" value="{{ "Reorder"|trans({}, 'Admin.Actions') }}" onclick="productOrderPrioritiesTable();" />
+                <input type="button" class="btn btn-outline-secondary" name="products_filter_position_asc" value="{{ "Reorganize"|trans({}, 'Admin.Actions') }}" onclick="productOrderPrioritiesTable();" />
```

Comment:
```suggestion
                <input type="button" class="btn btn-outline-secondary" name="products_filter_position_asc" value="{{ "Rearrange"|trans({}, 'Admin.Actions') }}" onclick="productOrderPrioritiesTable();" />
```

Reply:
After verification, I already see "Rearrange" in English, 1.7.8.5 🤔

Reply:
@Julievrz So do we still go with "Rearrange" ? Or another word needs to be chosen?

Reply:
"Rearrange" is ok, but I don't understand because when I check the BO in 1.7.8.5 I can already see the wording "Rearrange".

Reply:
@Julievrz I'm not sure what you mean

- if you open this back office page with 1.7.8.5 you see the wording being used is "Rearrange", not "Reorder" ?
- if you open the "Translation management" back office page with 1.7.8.5 you see the wording "Rearrange" is already registered which would mean it's already being used in the software?

Reply:
Update: the wording is in "Rearrange" because I put "Rearrange" as a translation 5 months ago. So I confirm we can go with Rearrange. Sorry for the misunderstanding.

Reply:
OK 👍